subsys/lorawan/services/Kconfig

```diff
@@ -54,4 +54,14 @@ config LORAWAN_APP_CLOCK_SYNC_PERIODICITY
 
 	  Default setting: 24h.
 
+config LORAWAN_REMOTE_MULTICAST
+	bool "Remote Multicast Setup"
+	depends on LORAWAN_APP_CLOCK_SYNC
+	depends on LORAWAN_NVM_SETTINGS
+	help
+	  Enables the LoRaWAN Remote Multicast Setup service according to
+	  TS005-1.0.0 as published by the LoRa Alliance.
+
+	  The service uses the default port 200.
```

Comment:
Uses the default multicast port, not the default port

Reply:
This would be confusing, as one could think the multicast messages are also sent on this port, which is not the case.

This Kconfig enables the LoRaWAN Remote Multicast Setup service and uses the default port 200 for this service. I think it's clear as it is. And the wording is exactly the same as for the already existing Application Layer Clock Sync service, btw.

Reply:
200 is not the "default port" any more than 202 is the "default port" because of `LORAWAN_PORT_CLOCK_SYNC`.
It's the default multicast port. I agree with the first sentence, so maybe "default downlink multicast port".

This can definitely be implied from the context that its in, but being explicit is rarely bad

Reply:
I find "default downlink multicast port" even more confusing. Multicast downlinks can be sent using any port. The FUOTA fragments are all sent as multicast downlink messages, but on port 201, which is the default for fragmented data. The multicast setup service which we are talking about actually only sends singlecast messages on that port 200.

We could say "The service uses port 200, which is the default port for the Remote Multicast Setup service.". But that repeats almost the entire sentence above.

I prefer keeping it as is.

Reply:
Cool, I agree with that. What about renaming to `LORAWAN_PORT_MULTICAST_SETUP`?

Reply:
Done.